src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/BotSegment.cs

```diff
@@ -20,11 +20,21 @@ public class BotSegment
 
         // versioning support for bot segments in the SDK, the is for this top level schema only
         // update this if this top level schema changes
-        public int apiVersion = SdkApiVersion.VERSION_1;
+        public int apiVersion = SdkApiVersion.VERSION_11;
```

Comment:
should this be `VERSION_12`?

Reply:
it will end up being version_13 i think after merge